docs/sources/review/lint-prose/index.md

```diff
@@ -79,22 +91,25 @@ If you are installing Vale on macOS, make sure that [Homebrew](https://brew.sh/)
 1. Download and install [Vale](https://vale.sh/docs/vale-cli/installation/).
 
    {{% admonition type="note" %}}
-   Refer to the Linux command line installation steps at [GitHub Releases](https://vale.sh/docs/vale-cli/installation/#github-releases). Verify that you are downloading the latest build of Vale for Linux.
-   You can also use your Linux package manager to install Vale.
+   [Verify](https://github.com/errata-ai/vale/releases) that you are downloading the latest build of Vale for Linux.
```

Comment:
I think it might be better to explain the reason for the link so people know whether to click it.
Maybe something like:

```suggestion
   Verify that you are downloading the latest build of Vale for Linux.
   To find the latest builds, refer to [Releases - errata-ai/vale](https://github.com/errata-ai/vale/releases).
```